client/CMT.cpp

```diff
@@ -847,7 +848,7 @@ static bool checkVideoMode(int monitorIndex, int w, int h, int& bpp, bool fullsc
 	const int modeCount = SDL_GetNumDisplayModes(monitorIndex);
 	for (int i = 0; i < modeCount; i++) {
 		SDL_GetDisplayMode(0, i, &mode);

```

Comment:
As a part of this refactoring I would suggest using `monitorIndex` variable, instead of hardcoding `0` here.